### ISSUE TITLE
material/form-field: outline appearance contrast does not meet WCAG ratio

### ISSUE THREAD

**Issue by ianbever** [angular/components#18730]
Regarding the accessibility built into Angular Material's components, I see that the default color for the outline form field is a 1px rgba(0,0,0,0.12) border that doesn't meet a 3:1 contrast ratio with a white surface to meet WCAG 2.1 Success Criterion 1.4.11: Non-text Contrast.

https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html

Is it possible to darken this to black at rgba(0,0,0,0.42), which would meet the minimum WCAG color contrast for non-text elements?

**Comment by devversion** [comment]
Thanks for this issue. The colors for the form-field are actually provided by the Material Design specification. We are also working on wrapping our form-field on the MDC text-field.

If you think that contrast is still an issue there, then it would be highly appreciated if you could submit an issue in the [material-componens-web](https://github.com/material-components/material-components-web) repository. Thank you!

MDC text-field seems to use a 2px `0,0,0,0.38` for the outline border. https://github.com/material-components/material-components-web/blob/master/packages/mdc-textfield/_variables.scss#L64
